client/my-sites/hosting/phpmyadmin-card/index.js

```diff
@@ -61,9 +62,21 @@ const PhpMyAdminCard = ( { translate, siteId, token, loading, disabled } ) => {
 			<CardHeading>{ translate( 'Database Access' ) }</CardHeading>
 			<p>
 				{ translate(
-					'Manage your database with phpMyAdmin and run a wide range of operations with MySQL.'
+					'For the tech-savvy, manage your database with phpMyAdmin and run a wide range of operations with MySQL.'
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Manage your database with phpMyAdmin and run a wide range of operations with MySQL.' )` *ES Score: 6*